code/datums/wounds/cranial_fissure.dm

```diff
@@ -87,6 +87,10 @@
 		return TRUE
 
 	victim.balloon_alert(user, "pulling out eyes...")
+	user.visible_message(
+		span_boldwarning("[user] reaches inside [victim]'s skull..."),
+		ignored_mobs = user
+	)
```

Comment:
Does `show_message` not do this same thing?

Reply:
Oh I guess not

Reply:
Use `balloon_alert_to_viewers` instead of this

Reply:
are balloon alerts usually used for something that you're not doing yourself? i feel like using a balloon alert here would be easily missed

Reply:
"Usually" no but there's no guideline against it--the actual eye pull is here, but the in progress doesn't need to be.

Reply:
alright

Reply:
Actually, no, not alright, i can't find a way where a balloon alert would be clear about what is going on while also being short. I think this should indeed be more like a combat notification